docs/resources/logs_custom_pipeline.md

```diff
@@ -1,275 +1,714 @@
--   user_agent_parser
-    -   `sources`: (Required) List of source attributes.
-    -   `target`: (Required) Name of the parent attribute that contains all the extracted details from the sources.
-    -   `is_encoded`: (Optional, default = false) If the source attribute is URL encoded or not.
-    -   `name`: (Optional) Name of the processor
-    -   `is_enabled`: (Optional, default = false) If the processor is enabled or not.
-
-Even though some arguments are optional, we still recommend you to state **all** arguments in the resource to avoid unnecessary confusion.
+## Schema
 
-## Import
+### Required
+
+- **filter** (Block List, Min: 1) (see [below for nested schema](#nestedblock--filter))
+- **name** (String, Required)
+
+### Optional
+
+- **id** (String, Optional) The ID of this resource.
+- **is_enabled** (Boolean, Optional)
+- **processor** (Block List) (see [below for nested schema](#nestedblock--processor))
+
+<a id="nestedblock--filter"></a>
+### Nested Schema for `filter`
+
+Required:
+
+- **query** (String, Required) Filter criteria of the category.
+
+
+<a id="nestedblock--processor"></a>
+### Nested Schema for `processor`
+
+Optional:
+
+- **arithmetic_processor** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--arithmetic_processor))
+- **attribute_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--attribute_remapper))
+- **category_processor** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--category_processor))
+- **date_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--date_remapper))
+- **geo_ip_parser** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--geo_ip_parser))
+- **grok_parser** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--grok_parser))
+- **lookup_processor** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--lookup_processor))
+- **message_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--message_remapper))
+- **pipeline** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--pipeline))
+- **service_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--service_remapper))
+- **status_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--status_remapper))
+- **string_builder_processor** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--string_builder_processor))
+- **trace_id_remapper** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--trace_id_remapper))
+- **url_parser** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--url_parser))
+- **user_agent_parser** (Block List, Max: 1) (see [below for nested schema](#nestedblock--processor--user_agent_parser))
```

Comment:
no description ?

Reply:
Good catch. All of these are missing descriptions...
The descriptions for these that I could copy from are pretty huge. WDYT of Stating the type and linking to the full docs like - https://docs.datadoghq.com/logs/processing/processors/

Reply:
Yeah sounds good to me